pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/pipeline/PipelineBreakerExecutor.java

```diff
@@ -68,7 +69,7 @@ private PipelineBreakerExecutor() {
   @Nullable
   public static PipelineBreakerResult executePipelineBreakers(OpChainSchedulerService scheduler,
       MailboxService mailboxService, WorkerMetadata workerMetadata, StagePlan stagePlan,
-      Map<String, String> opChainMetadata, long requestId, long deadlineMs) {
+      Map<String, String> opChainMetadata, long requestId, long deadlineMs, ThreadExecutionContext parentContext) {
```

Comment:
parentContext should be nullable.

nit/suggestion: Can we have a executePipelineBreaker(OpChainSchedulerService scheduler,
      MailboxService mailboxService, WorkerMetadata workerMetadata, StagePlan stagePlan,
      Map<String, String> opChainMetadata, long requestId, long deadlineMs) that just calls this method with null as last argument? That would reduce this PR by a large margin.

Reply:
Made the change. This was useful in `PipelineBreakerTest` only. In other usages, a variable is passed through and cannot be removed.